_data/banking.yml

```
@@ -234,6 +234,14 @@ websites:
     facebook: bbvacompass
     twitter: BBVACompass
 
+  - name: Bendigo Bank
+    url: https://www.bendigobank.com.au
+    img: bendigobank.png
+    facebook: BendigoBank 
```

Whoops, seems to be an extra space here 😅 
```suggestion
    facebook: BendigoBank
```